ui/src/App.scss

```diff
@@ -9,6 +9,10 @@
       margin-top: 0px !important;
       padding: $header-padding;
     }
+
+    @media (max-width: $screen-lg-max) {
+      padding: 0 1rem;
```

Comment:
maybe you could use the `xs` one already which is `12px` just `2` less than this.

Reply:
👍 done